docs/reference/compose.md

```diff
@@ -0,0 +1,86 @@
+
+
+## Description
+
+You can use compose subcommand, `docker compose [-f <arg>...] [options] [COMMAND] [ARGS...]`, to build and manage
+multiple services in Docker containers.
+
+### Use `-f` to specify name and path of one or more Compose files
+Use the `-f` flag to specify the location of a Compose configuration file.
+
+#### Specifying multiple Compose files
```

Comment:
@Usha, I seem to remember there are limitations in how many ## we support in docs tooling, is this OK ?